rpi_backlight.py

```diff
@@ -16,7 +16,8 @@
 
 __author__ = "Linus Groh"
 __version__ = "1.8.1"
-PATH = "/sys/class/backlight/rpi_backlight/"
+PATH = "/sys/devices/platform/ff150000.i2c/i2c-3/3-0045/"
```

Comment:
Is this the same across different boards and displays?

Reply:
This is how to control the Official 7" Raspberry Pi Display on the Asus Tinkerboard / Tinkerboard S

Reply:
That doesn't really answer my question :wink:
I'm just wondering if these three parts are guaranteed to always be the same: `ff150000.i2c`, `i2c-3` and `3-0045`.

Reply:
Referencing the Tinkerboard devs https://tinkerboarding.co.uk/forum/thread-2048-post-8814.html#pid8814
It says:

> Hi, mikeKey
   Can you try this path /sys/devices/platform/ff150000.i2c/i2c-3/3-0045/tinker_mcu_bl ?
   You can echo 0 ~ 255 into this node.
B/R
james

> 

So based on the devs response I think these will not change.

Reply:
Ok, that'll do. Thanks!